README.md

```diff
@@ -1,2 +1,105 @@
-# Math-Magicians
-This is the project of the Math Magicians calculator application. I set up the environment and tools needed to develop this app by a React.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math Magicians
+
+> This is the project of the Math Magicians calculator application. I set up the environment and tools needed to develop this app by a React.
+
+## Built With
+
+- javascript,react
+- github workflows
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+git clone git@github.com:sameh080081/Math-Magicians.git
```

Comment:
- [ ] OPTIONAL Please, consider fixing the README file: as you can see here the content is repeated twice 👍

Reply:
@elmar8287 thank you for your review "consider fixed"